Add optional settings (#2211)
zkSNACKs/WalletWasabi: WalletWasabi.Documentation/BackendDeployment.md

tail -10000 .walletwasabi/backend/Logs.txt
du -bsh .walletwasabi/backend/IndexBuilderService/*
```

> # Additional (optional) settings

Reviewer: ```suggestion
# Additional (optional) Settings
```

## Rolling Bitcoin Core node debug logs

> The following command line adds a configuration file to let logrotate service to know

Reviewer: ```suggestion
The following command line adds a configuration file to let logrotate service know
```

how to rotate the bitcoin debug logs.

> **Note:** in test server replaces the first line by the follow one `/home/user/.bitcoin/testnet3/debug.log`

Reviewer: ```suggestion
**Note:** In test server replace the first line by the following one `/home/user/.bitcoin/testnet3/debug.log`
```

```sh
sudo tee -a /etc/logrotate.d/bitcoin <<EOS
/home/user/.bitcoin/debug.log
{
        su user user
        rotate 5
        copytruncate
        daily
        missingok
        notifempty
        compress
        delaycompress
        sharedscripts
}
EOS
```

> ## Welcome banner

Reviewer: ```suggestion
## Welcome Banner
```
